assets/bingo/lang/pl_pl.json

```diff
@@ -1,12 +1,10 @@
 {
-
-
-
-
```

Comment:
Also... those lines got added back (probably also Crowdin)

Reply:
Yeah I marked them hidden on crowdin's side in the meantime, so next sync they should be gone again